api/optimeet/groups/models.py

```diff
@@ -32,3 +32,22 @@ class Recommendations(models.Model):
     place_url = models.CharField(max_length=100)
     times = models.JSONField()
     
+class Votes(models.Model):
+    
+    PREFERENCES_ACTIVE = "Preferences Active"
+    PREFERENCES_COMPLETE = "Preferences Complete"
+    RECOMMENDATION_ACTIVE = "Recommendation Active"
+    RECOMMENDATION_COMPLETE = "Recommendation Complete"
+    
+    voting_status_choices = [
+        (PREFERENCES_ACTIVE, "Preferences Active"),
+        (PREFERENCES_COMPLETE, "Preferences Complete"),
+        (RECOMMENDATION_ACTIVE, "Recommendation Active"),
+        (RECOMMENDATION_COMPLETE, "Recommendation Complete"),
+    ]
+
+    group_id = models.ForeignKey(Group, related_name='group_votes', on_delete=models.CASCADE)
+    max_capacity = models.IntegerField(blank=True)
+    voting_status = models.CharField(max_length=50, choices=voting_status_choices, default=PREFERENCES_ACTIVE)
+    results_of_voted = models.JSONField(blank=True)
```

Comment:
I don't think we need all these attributes, remember that these are relational models and we should use multiple rows whenever possible. We can have a single vote result in each row instead of a JSON field and then fetch them all by `group_id`. The status and max capacity are already a part of the group so I don't think we need to duplicate them here. We do need to keep track of who made this vote, so let's add a `user_id`. Remember you can always refer back to the schema in our API doc!